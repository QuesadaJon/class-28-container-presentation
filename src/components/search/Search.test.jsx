import React from 'react';
import { render } from '@testing-library/react';
import Search from './Search';

describe('Search component', () => {
  it('matches a snapshot', () => {
    const { asFragment } = render(
      <Search search="blah" onChange={jest.fn()}/>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
