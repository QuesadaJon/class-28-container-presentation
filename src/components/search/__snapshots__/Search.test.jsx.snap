// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component matches a snapshot 1`] = `
<DocumentFragment>
  <input
    class="Search"
    placeholder="Search"
    type="text"
    value="blah"
  />
</DocumentFragment>
`;
